stock_manual_transfer/views/stock_manual_transfer_views.xml

```diff
@@ -33,26 +33,26 @@
                         </h1>
                     </group>
                     <group col="4">
-                        <field name="warehouse_id" options="{'no_create': True}" />
-                        <field name="date_planned" />
-                        <field name="route_id" options="{'no_create': True}" />
+                        <field name="warehouse_id" options="{'no_create': True}" readonly="state != 'draft'" />
+                        <field name="date_planned" readonly="state != 'draft'" />
+                        <field name="route_id" options="{'no_create': True}" readonly="state != 'draft'" />
                     </group>
                     <notebook>
                         <page string="Transfer Lines" name="transfer_lines">
-                            <field name="transfer_line_ids" nolabel="1">
+                            <field name="transfer_line_ids" nolabel="1" readonly="state != 'draft'">
                                 <tree editable="bottom">
                                     <field name="sequence" widget="handle" />
                                     <field name="product_id" />
                                     <field name="product_uom_qty" />
-                                    <field name="product_uom_id" groups="uom.group_uom" optional="show" />
```

Comment:
This change and the one below are made because if the user don't belong to that group the compute of that field is not triggered, do you have any context why this happens in v17?